ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/TimeoutTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.jsonrpcproxy;
+
+import tech.pegasys.ethsigner.core.jsonrpc.response.JsonRpcError;
+import tech.pegasys.ethsigner.core.jsonrpc.response.JsonRpcErrorResponse;
+
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.json.Json;
+import org.junit.Test;
+import org.web3j.protocol.core.Request;
+import org.web3j.protocol.core.methods.response.EthProtocolVersion;
+
+public class TimeoutTest extends IntegrationTestBase {
+
+  @Test
+  public void downstreamConnectsButDoesNotRespondReturnsGatewayTimeout() {
+    final Request<?, EthProtocolVersion> jsonRpcRequest = jsonRpc().ethProtocolVersion();
```

Comment:
These two lines are common with another IT (`FailedConnectionTest. failsToConnectToDownStreamRaisesTimeout`), refactoring into a new class (e.g. `ProtocolVersion`) would nicely encapsulate the creation of the `String` payload and provide code reuse.

Reply:
Its done ... seems like a less than necessary change ...